server/controllers/user.js

```diff
@@ -209,12 +211,14 @@ module.exports = {
         messagePostedBy: req.user.id,
         messagePostDate: new Date(),
       });
-      await request.save();
+      const newMessage = await request.save();
```

Comment:
does **request.save()** return the new message?

Reply:
it doesn't, i'm just updating a part of state in the request messages